.github/workflows/build-wheels.yml

```diff
@@ -58,17 +71,19 @@ jobs:
           # different, and are containerised fully, therefore they get
           # their toolchain path from pyproject.toml
           CMAKE_TOOLCHAIN_FILE: ${{ github.workspace }}/.conan/conan_paths.cmake
-          # We matrix the python build here ourselves, rather than
-          # letting cibuildwheel do its regular python matrix, as we
-          # want to allow a full release pipeline to run "vertically" on
-          # CI, on individual agents, rather than building every python
-          # version sequentially on each platform. This is done less for
-          # performance reasons (although it does help) and more so a
-          # single failing python version won't interrupt every other
-          # deploy on that platform.
-          CIBW_BUILD: ${{ matrix.python-build }}
           CIBW_SKIP: '*musllinux* *arm64*'
-          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+          # Python 3.9 implies VFX Reference Platform CY22, which
+          # specifies glibc 2.17 and the deprecated libstdc++ ABI, and
```

Comment:
[question] I've not heard it called 'deprecated' before. Have you got a reference?

https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html does not mention deprecated.

I've heard 'new' and 'old' ABIs referred to.
It's not even you can say C++11 onward, as we used C++11 in the old ABI.

Reply:
That's just me, it feels wrong to say "new ABI" now that it's 9 years since it was introduced and RHEL/CentOS 7 is EOL :grimacing: To the best of my knowledge, the "old ABI"  its no longer the default anywhere.

If its going to raise eyebrows using the word "deprecated", then I can change the language.

Reply:
My main issue with deprecated is that I'm not sure it is. No idea how long it will hang around for.

With that link above they do say
> The _GLIBCXX_USE_CXX11_ABI macro controls whether the declarations in the library headers use the old or new ABI

so new/old is the documented wording from GNU.